lib/qwerty/crop.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'draw'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.